concepticondata/concepticon.tsv

```diff
@@ -489,7 +489,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 488	MANNER	Cognition	Way of performing or effecting anything.	Person/Thing	
 489	KIDNEY	The body	An organ in the body that produces urine.	Person/Thing	
 490	DESCENDANTS	Kinship	One who is the progeny of someone at any distance of time.	Person/Thing	
-491	CENTER	Spatial relations	The middle part, that can be clearly discerned from the enveloping part.	Person/Thing	
+491	CENTER	Spatial relations	What's in the mid of a plane or sphere.	Person/Thing	
```

Comment:
If it is person/thing referring to spatial relations, the original definition seems more appropriate. It should refer to the area/volume of something, not necessarily a plane or sphere (think about typical elicitations such as "what is in the center of the village?")

Reply:
I changed the original definition because explicitly refers to a center separate from the surroundings, not to the position in the center in general. By "plane or sphere" I meant that this "center" can be both in 2- and 3-dimensional space (whereas our "middle" is only in 1-dimensional space, e.g. in the middle of a row of things).

Reply:
Yes, this was clearer later with the MIDDLE concept. Maybe you should add a bit of these descriptions (1-dimensional, 2- or more) in the definitions? It might seem pedantic, but this is the kind of thing that helps people who are aligning conceptlist to concepticon.